tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,66 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2025.
 
+## August 12, 2025
+
+**General changes**
+
+- Rename "TiDB Cloud Serverless" to "**TiDB Cloud Starter**".
+
+    The auto-scaling entry plan is now named "TiDB Cloud Starter" to better reflect its role for new users. All features, pricing, and the free usage quota remain unchanged.
+
+    Starting August 12, 2025 (PDT), your existing Serverless clusters will appear as Starter in the [TiDB Cloud console](https://tidbcloud.com). Your connection strings, endpoints, and data will remain unchanged, so you do not need to make any code changes or schedule downtime.
+
+- TiDB Cloud Starter and **TiDB Cloud Essential** are in preview on Alibaba Cloud. 
+
+    TiDB Cloud Starter and TiDB Cloud Essential on Alibaba Cloud have been in a limited public preview since May 2025. This is the first time Essential on Alibaba Cloud is officially included in the release notes. At this stage, Essential on Alibaba Cloud offers a feature set aligned with Starter, available in the Alibaba Cloud Singapore region.
```

Comment:
```suggestion
    TiDB Cloud Starter and TiDB Cloud Essential on Alibaba Cloud have been in a limited public preview since May 2025. This is the first time Essential is officially included in the release notes. At this stage, Essential on Alibaba Cloud offers a feature set aligned with Starter, available in the Alibaba Cloud Singapore region.
```